src/nulog-inference-service/start-nulog-inference.py

```diff
@@ -94,38 +98,29 @@ async def doc_generator(df):
             df["nulog_confidence"] = predictions
             df["predictions"] = [1 if p < THRESHOLD else 0 for p in predictions]
             # filter out df to only include abnormal predictions
-            df = df[df["predictions"] > 0]
-            if len(df) == 0:
-                logging.info(
-                    "No anomalies in this payload of {} logs".format(len(masked_log))
-                )
-                continue
 
             df["_op_type"] = "update"
             df["_index"] = "logs"
-            df["script"] = (
-                "ctx._source.anomaly_predicted_count += 1; ctx._source.nulog_anomaly = true;"
-                + script
-            )
-
             df.rename(columns={"log_id": "_id"}, inplace=True)
-            df["script"] = (
-                df["script"]
-                + "ctx._source.nulog_confidence = "
-                + df["nulog_confidence"].map(str)
-                + ";"
-            )
-
+            df["script"] = [
```

Comment:
@sanjay920 FYI, this is where I fixed the inline script. 
basically the idea is - `script` has three fields: 
1. `source` for your script, but it should be a fix string rather than different strings in each call, such as `df["nulog_confidence"].map(str)`, then ES only needs to compile the source script once instead of many times
2. "lang" : "painless".
3. "params": the parameters that the value always changes, in my cases is the nulog_score value.